fedimintd/src/bin/main.rs

```diff
@@ -30,7 +30,7 @@ pub struct ServerOpts {
     #[arg(env = "FM_PASSWORD")]
     pub password: Option<String>,
     /// Port to run admin UI on
-    #[arg(long = "ui-bind", default_value = None)]
+    #[arg(long = "ui-bind", env = "FM_LISTEN_UI")]
```

Comment:
Nit: would be nice if the flag and environment variable had more similar name e.g. `--listen-ui` and `FM_LISTEN_UI`

Reply:
This is listed in https://github.com/fedimint/fedimint/issues/1251 . I didn't want to change anything existing just yet.